### PR TITLE
Clean packages to follow catkin_lint recommendations

### PR DESCRIPTION
**Pull request by ipa-nhg**
Solves: https://github.com/ros-industrial/industrial_core/issues/194

### CODE REVIEW

**Comment by mathias-luedtke** [comment]
Please consider adding `CATKIN_LINT=true`  (`pedantic` :sunglasses:) to the CI config as well :)

File: simple_message/CMakeLists.txt [hunk 1]
```diff
@@ -90,56 +87,58 @@ add_library(${PROJECT_NAME}_dummy ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_du
...
     LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-    RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
+    RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
 install(
-    DIRECTORY include/${PROJECT_NAME}/
-    DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
+    DIRECTORY include/${PROJECT_NAME}
+    DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+    RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```
Comment (gavanderhoorn) [inline simple_message/CMakeLists.txt]: Travis is failing on these lines. Are they necessary for `install(..)` commands that install headers? I was under the impression only libraries needed 'special care' like this (due to Windows using different paths for them, etc).

Comment (VictorLamoine) [reply simple_message/CMakeLists.txt]: I confirm it's not necessary!

Comment (mathias-luedtke) [reply simple_message/CMakeLists.txt]: >  I was under the impression only libraries needed 'special care' like this (due to Windows using different paths for them, etc).

I think this was fixed at some point, http://docs.ros.org/melodic/api/catkin/html/howto/format2/building_libraries.html#installing should be fine

Comment (mathias-luedtke) [reply simple_message/CMakeLists.txt]: `RUNTIME DESTINATION` is only valid for target-based install rules, please don't specify it for include directories

**Comment by gavanderhoorn** [comment]
Rebased version in #266.

Thanks for the initial PR @ipa-nhg :+1:

**Pull request closed without merging**